### ISSUE TITLE
Multiplied warnings at embedded webextension validation

### ISSUE THREAD

**Issue by vasilicamihasca** [mozilla/addons-linter#977]
**Steps to reproduce:**
1. Try to submit the following embedded webextension:
[Profilist-master.zip](https://github.com/mozilla/addons-linter/files/511536/Profilist-master.zip)
2. Select “See full validation report”

**Expected results:**
A warning is displayed only once.

**Actual results:**
A warning is displayed multiple times.

**Additional notes:**
- This issue reproduces on AMO-dev, AMO-stage and AMO-production using Firefox 52.0a1 (2016-10-05) under Windows 10 64-bit.
- Please see the attached screenshot for this issue:
  ![2016-10-05_1719](https://cloud.githubusercontent.com/assets/18114210/19119342/20885f3e-8b28-11e6-9024-e396f343698e.png)

**Comment by EnTeQuAk** [comment]
This is actually amo-validator stuff since it's an embedded webextension. I'll take a look at it, it's low priority though.

**Comment by andymckay** [comment]
@EnTeQuAk is there and amo-validator bug I can dupe this to?

**Comment by EnTeQuAk** [comment]
https://github.com/mozilla/amo-validator/issues/491
